Mathlib/Algebra/Category/ModuleCat/ChangeOfRings.lean

```diff
@@ -843,7 +843,7 @@ lemma extendRestrictScalarsAdj_homEquiv_apply
 lemma extendRestrictScalarsAdj_unit_app_apply
     {R : Type u₁} {S : Type u₂} [CommRing R] [CommRing S]
     (f : R →+* S) (M : ModuleCat.{max v u₂} R) (m : M) :
-    (extendRestrictScalarsAdj f).unit.app M m = (1 : S) ⊗ₜ[R,f] m :=
+    (extendRestrictScalarsAdj f).unit.app M m = (1 : S)⊗ₜ[R,f]m :=
```

Comment:
This one feels backwards (and doesn't seem to match the prevailing style in mathlib): let me revert from this PR.
```suggestion
    (extendRestrictScalarsAdj f).unit.app M m = (1 : S) ⊗ₜ[R,f] m :=
```